From efdc8dd4cb2e2d607f2c1cc0689faa6590c164f2 Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei@gherzan.ro>
Date: Wed, 3 Jul 2019 13:53:29 +0100
Subject: [PATCH 688/703] configs: arm64/bcm2711: Add MMC_SDHCI_IPROC

This driver is used in the device tree for the emmc2 node.

See #3032

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>
---
 arch/arm64/configs/bcm2711_defconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1036,6 +1036,7 @@ CONFIG_MMC_BCM2835_DMA=y
 CONFIG_MMC_BCM2835_SDHOST=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MMC_SDHCI_IPROC=y
 CONFIG_MMC_SPI=m
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
